src/server/acl/acl_commands_def.h

```diff
@@ -106,4 +107,57 @@ inline const std::vector<std::string> REVERSE_CATEGORY_INDEX_TABLE{
     "_RESERVED", "_RESERVED", "_RESERVED", "_RESERVED", "_RESERVED",  "_RESERVED",   "_RESERVED",
     "_RESERVED", "FT_SEARCH", "THROTTLE",  "JSON"};
 
+using BitfieldIndexPar =
+    std::pair<size_t /*index of family in the vector */, uint64_t /*bit index mask*/>;
+using CommandsIndexStore = absl::flat_hash_map<std::string, BitfieldIndexPar>;
+using RevCommandField = std::vector<std::string>;
+using RevCommandsIndexStore = std::vector<RevCommandField>;
+
+class CommandTableBuilder {
+ public:
+  CommandTableBuilder(CommandsIndexStore* index, RevCommandsIndexStore* rindex, size_t pos)
+      : index_(index), rindex_(rindex), pos_(pos) {
+    rindex_->push_back({});
+  }
+
+  friend CommandTableBuilder& operator|(CommandTableBuilder& builder, std::string name) {
+    (*builder.index_)[name] = {builder.pos_, 1ULL << (builder.bit_number_++)};
+    (*builder.rindex_)[builder.pos_].push_back(std::move(name));
+    // We should crash if somehow we end up feeding more than the bitfield size
+    // This won't happen in production, since our tests will fail and we will never
+    // merge something that violates this
+    CHECK_LT(++builder.bits_limit, size_t(64));
+    return builder;
+  }
```

Comment:
Whats the difference between the builder and a single function that accepts an absl::Span<string_view> 😆 

But as I suggested above, lets not duplicate everything a third time. Lets finally make the command registry itself usable and then you don't need 800 lines of changes to do this all